src/pretix/base/forms/questions.py

```diff
@@ -264,17 +264,13 @@ class WrappedPhonePrefixSelect(Select):
 
     def __init__(self, initial=None):
         choices = [("", "---------")]
-        language = get_babel_locale()  # changed from default implementation that used the django locale
-        locale = Locale(translation.to_locale(language))
+
         for prefix, values in _COUNTRY_CODE_TO_REGION_CODE.items():
```

Comment:
Thanks to your optimization and removing the country_code bit, we could skip this for-loop entirely if initial is None. Furthermore we could `break` after setting self.initial. I could not make a code suggestion as indenting the loop would go over a removed line. If you want, I can add the change as a whole commit myself. Here is what I would suggest:

```
    def __init__(self, initial=None):
        choices = [("", "---------")] + get_phone_prefixes_sorted_and_localized()
        if initial:
            for prefix, values in _COUNTRY_CODE_TO_REGION_CODE.items():
                if initial in values:
                    self.initial = "+%d" % prefix
                    break

        super().__init__(choices=choices, attrs={'aria-label': pgettext_lazy('phonenumber', 'International area code')})

```